portfolio/src/main/webapp/index.html

```diff
@@ -36,7 +37,7 @@ <h2>About</h2>
             <p>And here are some movies I love:</p>
             
         </div>
-        <div id="movie-container"><body onload="onLoad()" id="movies" class="section"></div><br>
+        <div id="movie-container"><body id="movies" class="section"></div><br>
```

Comment:
was it intentional to add a body tag here?

Reply:
This section wasn't showing up properly before so I added the body tag here intentionally, but later found out that the problem was something else.